<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDataDialog</class>
 <widget class="QDialog" name="UserDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Renseigner les données vous concernant</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../icons.qrc">
    <normaloff>:/icons/FactDev</normaloff>:/icons/FactDev</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="1">
    <widget class="QPushButton" name="btnValid">
     <property name="text">
      <string>Sauvegarder</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons.qrc">
       <normaloff>:/icons/img/ok_valid.png</normaloff>:/icons/img/ok_valid.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>Annuler</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons.qrc">
       <normaloff>:/icons/img/cancel.png</normaloff>:/icons/img/cancel.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QTabWidget" name="tabUserData">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab1PersonalData">
      <attribute name="title">
       <string>Données personnelles</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lbLastname">
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="lbMobilePhone">
         <property name="text">
          <string>Tél. mobile</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="lbNoSiret">
         <property name="text">
          <string>N° Siret</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="CheckSiretNumber" name="leNoSiret" native="true">
         <property name="placeholderText" stdset="0">
          <string>123 123 123 1234</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="CheckPhone" name="leMobilePhone" native="true">
         <property name="placeholderText" stdset="0">
          <string>06 01 02 03 04</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Adresse</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="3">
           <widget class="CheckCity" name="leCity" native="true">
            <property name="placeholderText" stdset="0">
             <string>Paris</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lbAddress">
            <property name="text">
             <string>N°, Rue</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="lbCity">
            <property name="text">
             <string>Ville</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="CheckPostalCode" name="lePostalCode" native="true">
            <property name="placeholderText" stdset="0">
             <string>75000</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="CheckUntilField" name="leAddress" native="true">
            <property name="placeholderText" stdset="0">
             <string>42 Avenue de Toulouse</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbPostalCode">
            <property name="text">
             <string>Code Postal</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lbEmail">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="CheckEmail" name="leEmail" native="true">
         <property name="placeholderText" stdset="0">
          <string>remi.adelin@fiducal.com</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="lbPhone">
         <property name="text">
          <string>Tél. fixe</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="CheckPhone" name="lePhone" native="true">
         <property name="placeholderText" stdset="0">
          <string>01 02 03 04 05</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="CheckValidField" name="leTitle" native="true">
         <property name="placeholderText" stdset="0">
          <string>Expertise comptable</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lbCompany">
         <property name="text">
          <string>Société</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="CheckValidField" name="leCompany" native="true">
         <property name="placeholderText" stdset="0">
          <string>Fiducal8</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lbTitle">
         <property name="text">
          <string>Activité</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbFirstname">
         <property name="text">
          <string>Prénom</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="CheckName" name="leFirstname" native="true">
         <property name="placeholderText" stdset="0">
          <string>Rémi</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="CheckName" name="leLastname" native="true">
         <property name="toolTip">
          <string/>
         </property>
         <property name="placeholderText" stdset="0">
          <string>ADELIN</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>leTitle</zorder>
      <zorder>lbPhone</zorder>
      <zorder>lbLastname</zorder>
      <zorder>lbNoSiret</zorder>
      <zorder>groupBox</zorder>
      <zorder></zorder>
      <zorder>lbTitle</zorder>
      <zorder>lePhone</zorder>
      <zorder>lbMobilePhone</zorder>
      <zorder>lbEmail</zorder>
      <zorder>leEmail</zorder>
      <zorder>lbCompany</zorder>
      <zorder>leNoSiret</zorder>
      <zorder>leCompany</zorder>
      <zorder>leMobilePhone</zorder>
      <zorder>lbPhone</zorder>
      <zorder>lbLastname</zorder>
      <zorder>lbCompany</zorder>
      <zorder>leNoSiret</zorder>
      <zorder>groupBox</zorder>
      <zorder>lePhone</zorder>
      <zorder>leCompany</zorder>
      <zorder>leTitle</zorder>
      <zorder>lbEmail</zorder>
      <zorder>lbNoSiret</zorder>
      <zorder>lbTitle</zorder>
      <zorder>leMobilePhone</zorder>
      <zorder>lbMobilePhone</zorder>
      <zorder>leEmail</zorder>
      <zorder>lbFirstname</zorder>
      <zorder>leFirstname</zorder>
      <zorder>leLastname</zorder>
     </widget>
     <widget class="QWidget" name="tab2Workspace">
      <attribute name="title">
       <string>Environnement de travail</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Répertoire</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="1">
           <widget class="QLineEdit" name="leWorkspacePath">
            <property name="placeholderText">
             <string>Chemin d'accès </string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbWorkspacePath">
            <property name="text">
             <string>Chemin</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lbWorkspaceName">
            <property name="text">
             <string>Nom</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="btnBrowseWorkspace">
            <property name="text">
             <string>Parcourir</string>
            </property>
            <property name="icon">
             <iconset resource="../../icons.qrc">
              <normaloff>:/icons/img/folder.png</normaloff>:/icons/img/folder.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="leWorkspaceName">
            <property name="placeholderText">
             <string>Nom de votre espace de travail</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckPhone</class>
   <extends>QWidget</extends>
   <header>checkphone.h</header>
  </customwidget>
  <customwidget>
   <class>CheckSiretNumber</class>
   <extends>QWidget</extends>
   <header>checksiretnumber.h</header>
  </customwidget>
  <customwidget>
   <class>CheckValidField</class>
   <extends>QWidget</extends>
   <header>checkvalidfield.h</header>
  </customwidget>
  <customwidget>
   <class>CheckName</class>
   <extends>QWidget</extends>
   <header>checkname.h</header>
  </customwidget>
  <customwidget>
   <class>CheckCity</class>
   <extends>QWidget</extends>
   <header>checkcity.h</header>
  </customwidget>
  <customwidget>
   <class>CheckPostalCode</class>
   <extends>QWidget</extends>
   <header>checkpostalcode.h</header>
  </customwidget>
  <customwidget>
   <class>CheckUntilField</class>
   <extends>QWidget</extends>
   <header>checkuntilfield.h</header>
  </customwidget>
  <customwidget>
   <class>CheckEmail</class>
   <extends>QWidget</extends>
   <header>checkemail.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leAddress</tabstop>
  <tabstop>lePostalCode</tabstop>
  <tabstop>leCity</tabstop>
 </tabstops>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>leMobilePhone</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leMobilePhone</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leMobilePhone</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UserDataDialog</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>721</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leCompany</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leCompany</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leCompany</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UserDataDialog</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leNoSiret</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leNoSiret</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leNoSiret</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UserDataDialog</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leEmail</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leEmail</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leEmail</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UserDataDialog</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>589</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leLastname</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leLastname</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leLastname</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UserDataDialog</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePhone</sender>
   <signal>textChanged(QString)</signal>
   <receiver>lePhone</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePhone</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UserDataDialog</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>661</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leFirstname</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leFirstname</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leFirstname</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UserDataDialog</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>570</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnValid</sender>
   <signal>released()</signal>
   <receiver>UserDataDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>released()</signal>
   <receiver>UserDataDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePostalCode</sender>
   <signal>textChanged(QString)</signal>
   <receiver>lePostalCode</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leCity</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leCity</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leAddress</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leAddress</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leAddress</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UserDataDialog</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leCity</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UserDataDialog</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>595</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePostalCode</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UserDataDialog</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>617</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leTitle</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leTitle</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leTitle</sender>
   <signal>textChanged(QString)</signal>
   <receiver>UserDataDialog</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>fieldTextChanged(QString)</slot>
  <slot>checkFields()</slot>
 </slots>
</ui>
